chapter_03.md

```diff
@@ -61,6 +72,16 @@ If you prefer to write your tests with TypeScript or Babel you can pick one of t
 
 Since we've already written our automation script, we can press `n` here as our objective will be to transfer that script into a test.
```

Comment:
```suggestion
Since we've already written our automation script, we can press `n` here. We'll reserve the task of crafting our own page objects for a subsequent chapter.
```